assets/samplesheet.csv

```diff
@@ -1,3 +1,14 @@
-sample,fastq_1,fastq_2
```

Comment:
I actually only put it in there before making the pull request so that you could see it. 

And I haven't put it in the readme yet, but I will! Have mostly been focusing on getting this to work at all. Lots of housekeeping to do still